docs/circuit-chat.md

```diff
@@ -0,0 +1,87 @@
+# circuit-chat
+
+## Usage
+
+```html
+// Circuit Conversation. Renders the conversation feed of a given conversation Id.
```

Comment:
Why "Circuit Conversation"?

I would write: "Render text messages of conversation with an input field to post messages"